documentation/C++style.md

```diff
@@ -15,7 +17,15 @@ is clear on usage, follow it.
 * Reasonable exceptions to these guidelines can be made.
 * Be aware of some workarounds for known issues in older C++ compilers that should
   still be able to compile f18. They are listed at the end of this document.
+
 ## In particular:
+
+Use serial commas in comments, error messages, and documentation.
```

Comment:
I'd guess that people in general will not get the reference to "serial commas" and understand what it means.  How about "In a comma separated list, place a comma after the next-to-last item and the 'and' or 'or' that precedes the last item."

Reply:
That's why I included an example.